File: Java-Core/src/main/java/presto/grammar/CmpOp.java
package presto.grammar;


public enum CmpOp {
	GT(">"),
	GTE(">="),
	LT("<"),
	LTE("<=");

	String token;
	
	CmpOp(String token) {
		this.token = token;
	}
	
	public String toString() {
		return token;
	}
}

File: Java-CodeServer/src/main/resources/js/prompto/grammar/CmpOp.js
function CmpOp() {
	return this;
}

CmpOp.prototype.toDialect = function(writer) {
    writer.append(this.toString());
};

CmpOp.GT = new CmpOp();
CmpOp.GT.toString = function() {
    return ">";
}

CmpOp.GTE = new CmpOp();
CmpOp.GTE.toString = function() {
    return ">=";
}

CmpOp.LT = new CmpOp();
CmpOp.LT.toString = function() {
    return "<";
}

CmpOp.LTE = new CmpOp();
CmpOp.LTE.toString = function() {
    return "<=";
}

exports.CmpOp = CmpOp;
